metadata-sheets-schema/radiomics-metadatasheet-schema.json

```diff
@@ -3,20 +3,19 @@
     "$id": "https://raw.githubusercontent.com/qbicsoftware/",
     "title": "Radiomics metadata sheet",
     "description": "A Metadata object representation for mapping the radiomics metadata sheet to OpenBIS metadata fields.",
-    "definitions": {
+    "type": "object",
+    "properties": {
         "Filenames": {
-            "type": "column",
-            "properties": {
-                "label": {
-                    "type": "string"
-                },
-                "value": {
-                    "type": "string"
-                }
-            },
-            "required": ["label","value"],
-            "OpenBIS_intance": "Q_BMI_IMAGING_DATA",
+            "description": "File name / directory name of the tomogram entry.",
+            "type": "string",
+            "OpenBIS_instance": "Q_BMI_IMAGING_DATA",
```

Comment:
I am not sure if you can just state these OpenBIS_* Terms. I think you need to define them on the top. Like "definitions":{ 
"OpenBIS_instance":{ 
"type":"string"},
"OpenBIS_property":{
"type":"string"}
}
maybe we should ask @sven1103 if I am right

Reply:
And after the definition you can state them as you did